### PR TITLE
Add fully functional checkboxes, ordering system and localStorage

### PR DESCRIPTION
**Pull request by Zibilyonik**
-Add checkbox functionality to allow users to keep track of their completed tasks.
-Add dragging.js module to allow users to drag and organise their tasks.
-Add array reordering to keep the app stable.
-Add localStorage to keep the array stored for further use.

### CODE REVIEW

File: src/dragging.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+function dragger(arr) {
+  const item1 = arr[held];
+  const item2 = arr[droppedItem];
+  arr.splice(item1.index, 1);
+  arr.splice(item2.index, 0, item1);
+  for (let i = 0; i < arr.length; i += 1) {
+    arr[i].index = i;
+  }
+}
```
Comment (vichuge) [inline src/dragging.js:20]: - [ ] It looks like you have the dragger function, but in element, I couldn't be able to see where to change the order of the elements, I kindly invite you to do it. 🐱 
image

File: src/changeStatus.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const changeStatus = (arr, itemId) => {
+  const item = document.getElementById(itemId);
+  arr[itemId].completed = !arr[itemId].completed;
+  if (arr[itemId].completed) {
+    item.classList.add('disabled');
+  } else {
+    item.classList.remove('disabled');
+  }
+};
```
Comment (vichuge) [inline src/changeStatus.js:9]: - [ ] I could see you implement the change status function, and I'm able to check/uncheck the elements, one important detail here is to use `localStorage` to save the new info and show it when I refresh the page. At this point when I check/uncheck an element the application doesn't save the change. This is required for drag and drop too. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL] Considering using a better style for the project, more according the video (background color too).
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/changeStatus.js (hunk 2 above)
Comment (Davidosky007) [reply src/changeStatus.js]: * done 👍

**Comment by Davidosky007** [comment]
Hi @Zibilyonik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**